platforms/software/dependency-management/src/main/java/org/gradle/internal/component/resolution/failure/describer/ConfigurationDoesNotExistFailureDescriber.java

```diff
@@ -50,9 +50,4 @@ private String buildFailureMsg(ConfigurationDoesNotExistFailure failure) {
             failure.getTargetComponent().getDisplayName()
         );
     }
-
-    private String quoteNameOnly(String formattedId) {
```

Comment:
❔  was this unused?

Reply:
It was, I started to remove the `UnusedMethod` in this project, but decided that was to much for this PR now.
That is a leftover of this effort.
But I decided it is so obvious I keep this.